### ISSUE TITLE
Snap Package for Linux

### ISSUE THREAD

**Issue by codeman6100** [Thomas101/wmail#208]
It would be nice to have a snap package for use on linux devices.

**Comment by jadenPete** [comment]
Or a deb package

**Comment by Hronom** [comment]
++ for ubuntu deb

**Comment by luispabon** [comment]
Snap would be better as it'd be portable across any snapd supported distros. Package once, run anywhere!

**Comment by jadenPete** [comment]
Yes but many distros still don't support snap, i'm running Ubuntu 14.04.4

**Comment by luispabon** [comment]
Fair play, but the binaries already work on Ubuntu 14.04 as they're distributed today.

**Comment by almejo** [comment]
If you want a deb package open a issue for that. This is the issue for a snap :D

**Comment by Thomas101** [comment]
I think it would be great to support both! Would be super happy to accept a pull request for this if anyone wants to fast-track it :-D

**Comment by luispabon** [comment]
Maybe I can help with this actually.

**Comment by funilrys** [comment]
An Arch package has been done ;) 

`yaourt -S wmail-bin`
